src/functionaltests/java/com/ericsson/ei/utils/EventManager.java

```diff
@@ -38,15 +35,24 @@
      * @throws IOException
      */
     public void sendEiffelEvents(String eiffelEventsJsonPath, List<String> eventNamesToSend) throws IOException {
-        eventsIdList = new ArrayList<>();
         List<String> eventNames = eventNamesToSend;
         JsonNode parsedJSON = getJSONFromFile(eiffelEventsJsonPath);
         for (String eventName : eventNames) {
             JsonNode eventJson = parsedJSON.get(eventName);
-            eventsIdList.add(eventJson.get("meta").get("id").toString().replaceAll("\"", ""));
             rmqHandler.publishObjectToWaitlistQueue(eventJson.toString());
         }
     }
+    
+    public List<String> getEventsIdList(String eiffelEventsJsonPath, List<String> eventNamesToSend) throws IOException {
```

Comment:
A suggestion would be to simply return the eventsIdList on the sendEiffelEvents method.